src/server/coprocessor/endpoint.rs

```diff
@@ -486,14 +479,146 @@ fn get_chunk(chunks: &mut Vec<Chunk>) -> &mut Chunk {
     chunks.last_mut().unwrap()
 }
 
+#[inline]
+fn encode_row_from_cols(cols: &[ColumnInfo],
+                        h: i64,
+                        values: HashMap<i64, &[u8]>,
+                        data: &mut Vec<u8>)
+                        -> Result<RowMeta> {
+    let mut meta = RowMeta::new();
```

Comment:
yes,I added the test case: select * from table order by A desc,B asc limit 5, with some A is none.